src/clipboard-action.js

```diff
@@ -130,6 +130,7 @@ class ClipboardAction {
         else {
             this.emitter.emit('error', {
                 action: this.action,
+                text: this.selectedText,
```

Comment:
Assuming your change is correct, the object being emitted is the same for a success or error.  I think it would make sense to define this object once before the `if` and then use it in both places.

```javascript
let event = {
    action: this.action,
    text: this.selectedText,
    trigger: this.trigger,
    clearSelection: this.clearSelection.bind(this)
}
if (succeeded) {
    this.emitter.emit('success', event);
} else {
    this.emitter.emit('error', event);
}
```

An alternative could be:

```javascript
this.emitter.emit(succeeded ? 'success' : 'error', {
    action: this.action,
    text: this.selectedText,
    trigger: this.trigger,
    clearSelection: this.clearSelection.bind(this)
});
```